Riot/Modules/Common/Recents/DataSources/RecentsDataSource.m

```diff
@@ -966,6 +966,10 @@ - (UIView *)viewForHeaderInSection:(NSInteger)section withFrame:(CGRect)frame in
 
         if (self.allChatsOptionsView)
         {
+            [self.allChatsFilterOptions updateWithFilterOptionListView:self.allChatsOptionsView
+                                                          unreadsCount:1 // unreads is allways visible
```

Comment:
I'm wondering if it even makes sense to have the `unreadsCount` parameter on the method if we always pass `1` to force a result?

Reply:
It's clearly a matter of point of view. In my opinion, the `AllChatsFilterOptions` instance has to manage 3 tabs for now:
`unreads`, `favourites`, and `people`. For consistency purpose, this method needs the count for all those 3 tabs. Up to the caller to set the `unreads` tab as always visible as requested for now in the related issue.

I can change it if this makes no sense to you.

Reply:
Nah its totally fine as is, was only a thought :)